examples/workspace/WORKSPACE

```diff
@@ -1,5 +1,3 @@
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Move this into the comment below so that the instructions are still correct.

Reply:
Good call. Applied.